test/unit-test/test-Model.js

```diff
@@ -0,0 +1,851 @@
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Int32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_FLOAT32 (3), setting its value as a Uint32Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_FLOAT32 (3), setting its value as a Float32Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Float32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_FLOAT32 (3), setting its value as a Float64Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Float64Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_FLOAT32 (3), setting its value as a DataView is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let buffer = new ArrayBuffer(4);
+        let inputData = new DataView(buffer, 0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_INT32 (4), setting its value as a Int8Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_INT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Int8Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is TENSOR_INT32 (4), setting its value as a Uint8Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.TENSOR_INT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint8Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
```

Comment:
It should throw as wrong typed array type. and same applied for following tests.